CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.4.1
```

Comment:
Can you revert it? I prefer to write changes in release commit.

Reply:
+